obspy/clients/fdsn/client.py

```diff
@@ -1890,14 +1885,12 @@ def download_url(url, opener, timeout=10, headers={}, debug=False,
         url_obj = opener.open(request, timeout=timeout, data=data)
     # Catch HTTP errors.
     except urllib_request.HTTPError as e:
+        error_data = e.read()
```

Comment:
I'm confused about this one.. I don't see that `HTTPError` even has a `.read()` method.. :thinking: looks like `urllib` is adding some magic stuff if a filepointer is given to `HTTPError.__init__()` and then those additionally populated methods are left out in the docs..  but hey it works :smile: